events/templates/events/event_details.html

```diff
@@ -5,12 +5,11 @@
 </tr><tr>
 	<td><b>{% trans "Place:" %}</b></td><td>{{ event.place }}</td>
 <tr>
-	<td><b>{% trans "Host:" %}</b></td><td>{{ event.created_by }}</td>
+	<td><b>{% trans theme.text_labels.host %}</b></td><td>{{ event.created_by }}</td>
```

Comment:
@georgesco94 , on the template files, you don't have to add the translations keyword, since the settings.py file already indicates that the string should be translated. Please, can you remove the {% trans %} from the template files? 
```suggestion
	<td><b>{{ theme.text_labels.host }}</b></td><td>{{ event.created_by }}</td>
```

Reply:
@thiagoferreiraw ah correct, will do